### PR TITLE
Kotlin gradle starter with Spek testing.

### PR DESCRIPTION
**Pull request by vjames19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+
+class App : Kooby({
+    get {
+        val name = param("name").value("Kotlin")
+        "Hello $name!"
+    }
+})
+
+fun main(args: Array<String>) {
+    Jooby.run(::App, args)
```
Comment (jknack) [inline README.md]: there is a `run` function in 1.1.3: `run(::App, args)`

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+task wrapper(type: Wrapper) {
+    gradleVersion = "3.5"
+    distributionType = org.gradle.api.tasks.wrapper.Wrapper.DistributionType.ALL
+}
+
+dependencies {
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jre8:$kotlinVersion"
+    compile "org.jooby:jooby-lang-kotlin:$joobyVersion"
+    compile "org.jooby:jooby-netty:$joobyVersion"
```
Comment (jknack) [inline build.gradle]: what OS are you? I think we need some native dependencies for unix/linux users.

do you know a gradle plugin that detects OS?

File: src/main/kotlin/starter/kotlin/App.kt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+class App : Kooby({
+    get {
+        val name = param("name").value("Kotlin")
+        "Hello $name!"
+    }
+})
+
+fun main(args: Array<String>) {
+    Jooby.run(::App, args)
```
Comment (jknack) [inline src/main/kotlin/starter/kotlin/App.kt]: same here, let's use the `run` function

File: build.gradle (hunk 2 above)
Comment (vjames19) [inline build.gradle]: I'm on mac os.

Here is a plugin: https://github.com/google/osdetector-gradle-plugin

Is this for the netty dependencies?

File: src/main/kotlin/starter/kotlin/App.kt (hunk 3 above)
Comment (vjames19) [inline src/main/kotlin/starter/kotlin/App.kt]: Done

File: README.md (hunk 1 above)
Comment (vjames19) [inline README.md]: Done

File: build.gradle (hunk 2 above)
Comment (jknack) [inline build.gradle]: Yea, netty: https://github.com/jooby-project/jooby/blob/master/modules/jooby-netty/pom.xml#L153-L208

Comment (vjames19) [inline build.gradle]: @jknack I'm kinda unsure of what you want to do.

Do you publish jooby-netty with an OS classifier?

Or do you want the build.gradle to bring the right io.netty dependency depending on the OS?

**Pull request merged**